spec/factories/ticket_purchases.rb

```diff
@@ -4,5 +4,11 @@
     conference
     ticket
     quantity 10
+    paid false
+    factory :paid_ticket_purchase do
+    	after(:build) do |ticket_purchase|
+    		ticket_purchase.paid = true
```

Comment:
And BTW, normally overriding just an attribute is better to put this directly on the subfactory, not in a **after(:build)**:

```ruby
factory :paid_ticket_purchase do
  paid true
end
```